etc/config-template.toml

```diff
@@ -1104,6 +1104,14 @@
 ## Default: false.
 # enable-log-recycle = false
 
+## Whether to prepare log files for recycling when start.
+## If `true`, batch empty log files will be prepared for recycling when
+## starting engine.
+## Only available for `enable-log-reycle` is true.
+##
+## Default: false
+# prefill-for-recycle = false
```

Comment:
Should it have the test code to ensure the config passed to raft-engine correctly or behave correctly.

Reply:
Nope, it's unnecessary as these cases have already been covered by the related uts in `raft-engine` lib.